prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2016,6 +2017,47 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 		dlist_remove(&read_entry->pending_entry);
 	}
 
+	/*
+	 * Send remote read requests until finish or error encoutered
+	 */
+	dlist_foreach_container_safe(&ep->op_entry_queued_read_list, struct rxr_op_entry,
```

Comment:
before this change where do we progress queued read entry?

Reply:
Also in `rxr_ep_progress`, when we process `read_pending_list`.

https://github.com/ofiwg/libfabric/blob/a0ad62c76f9c00ecc6a9c17651ea16aba5328043/prov/efa/src/rdm/rxr_ep.c#L2003

This patch did not delete that code block, because local read still uses it.